src/Verilator.cpp

```diff
@@ -165,6 +165,7 @@ static void process() {
 
         // Remove any modules that were parameterized and are no longer referenced.
         V3Dead::deadifyModules(v3Global.rootp());
+        // V3Dead::deadifyModuleDTypes(v3Global.rootp());
```

Comment:
Oh it's not two passes. I wanted to use deadifyModuleDTypes instead of deadifyModules to clean-up modules & types, but after I found out it's not working as I expect, I switched back the annotation...